build.js

```diff
@@ -466,131 +699,173 @@ function executeOnCreate(api, app, config, opts) {
 
           if (retVal) { resolve(retVal); }
         })
-    });
-}
-
-function makeAndroidProject(api, app, config, opts) {
-  var projectPropertiesFile = path.join(opts.outputPath, 'project.properties');
-  return fs.unlinkAsync(projectPropertiesFile)
-    .catch(function () {}) // ignore error if file doesn't exist
-    .then(function () {
-      return spawnWithLogger(api, 'android', [
-          "create", "project", "--target", ANDROID_TARGET, "--name", app.manifest.shortName,
-          "--path", opts.outputPath, "--activity", config.activityName,
-          "--package", config.packageName
-        ])
-        .catch(BuildError, function (err) {
-          if (err.stdout && /not valid/i.test(err.stdout)) {
-            logger.log(chalk.yellow([
-                '',
-                'Android target ' + ANDROID_TARGET + ' was not available. Please ensure',
-                'you have installed the Android SDK properly, and use the',
-                '"android" tool to install API Level ' + ANDROID_TARGET.split('-')[1] + '.',
-                ''
-              ].join('\n')));
-          }
+      });
+  }
 
-          if (err.stdout && /no such file/i.test(err.stdout) || err.code == 126) {
-            logger.log(chalk.yellow([
-                '',
-                'You must install the Android SDK first. Please ensure the ',
-                '"android" tool is available from the command line by adding',
-                'the sdk\'s "tools/" directory to your system path.',
-                ''
-              ].join('\n')));
-          }
+  var projectPath = '';
+  function makeAndroidProject(api, app, config, opts) {
+    projectPath = path.resolve(__dirname+"/../../../../build/"+app.manifest.shortName);
```

Comment:
why cant we use build/native-android (opts.outputPath) as project path ?
why do we need build/shortName ?

Reply:
This task is in todos, please add your comments and improvements here.
https://github.com/CodeCombiner/native-android/issues/27